src/styles/blocks/page.scss

```diff
@@ -0,0 +1,21 @@
+@include font;
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't set margins and padding for all elements, reset only for elements that you actually need, for example div and span doesn't have any of them